graylog2-web-interface/src/pages/SidecarStatusPage.jsx

```diff
@@ -25,6 +26,7 @@ class SidecarStatusPage extends React.Component {
 
   componentDidMount() {
     this.reloadSidecar();
+    this.reloadCollectors();
```

Comment:
Do we also need to automatically reload collectors, or is it fine to only load them on page load? I guess the second option is fine, but want to double-check :)

Reply:
Loading them once is good enough.